os_migrate/tests/unit/test_security_group.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import (absolute_import, division, print_function)
```

Comment:
Cool that we have basic unit tests. This patch is getting rather big, do we want to add functional tests in a separate one? It would make repeated reviewing much easier if we merged this big patch. All in all the code looks very good so once the minor issues i listed above are addressed, i think we should just merge and then continue in smaller PRs? The Rules import/export is not usable yet because missing security_group_name in serialized data, but that's fine.